packages/smart-router/evm/stableSwap/getLPOutput.ts

```diff
@@ -44,13 +36,13 @@ export function getLPOutput({
   for (const [i, balance] of balances.entries()) {
     const amount = amounts[i] || CurrencyAmount.fromRawAmount(balance.currency, 0)
     invariant(
-      amount.currency.wrapped.equals(balance.currency.wrapped),
+      amount.currency.equals(balance.currency),
```

Comment:
need wrap for native currency passing in

Reply:
done